### ISSUE TITLE
Contao 5.x Support

### ISSUE THREAD

**Issue by itnomic** [chopsol/contao-matomo-tracker#5]
Hallo! Leider ist das Modul nicht mit Contao 5.x kompatibel. Ist ein Update geplant?

**Comment by chopsol** [comment]
Ja, es ist geplant, die Erweiterung auch für Contao 5 anzubieten, allerdings kann ich derzeit noch kein Zeitfenster nennen. Ich hoffe es bis zum Release der 5.3 LTS spätestens umgesetzt zu haben.

**Comment by codesache** [comment]
@chopsol Wie schaut es mit der Anpassung für 5.3.x aus ?

**Comment by cmeis** [comment]
Ja, 5.x-Kompatibilität wäre ECHT super. Ist absehbar, dass da noch was kommt? ;-)

**Comment by chopsol** [comment]
Ja, es wird eine angepasste Version kommen. Wenn die Planung standhält, werde ich es im Mai 2024 schaffen. Ich halte euch hier auf dem laufenden.
